examples/example_xyz.py

```diff
@@ -0,0 +1,92 @@
+from countoscope import Countoscope
+import matplotlib.pyplot as plt
+import numpy as np
+
+"""
+run this example from within the examples folder with
+python example_xyz.py
+"""
+
+# this minimal working example currently uses a shortened copy of datasets/brownian-particles/2D-closed/trajectory.xyz
+
+# set up the Countoscope
+countoscope = Countoscope(
+    trajectory_file  = 'trajectory2.xyz',
```

Comment:
should it be trajectory_file  = 'trajectory.xyz' (without the 2)

Reply:
oops, thanks